browser/ui/commander/commander_service_browsertest.cc

```diff
@@ -179,7 +179,7 @@ IN_PROC_BROWSER_TEST_F(CommanderServiceBrowserTest,
   auto items = commander()->GetItems();
   ASSERT_EQ(3u, items.size());
   EXPECT_EQ(u"Pin tab", items[0].title);
-  EXPECT_EQ(u"Pin tab...", items[1].title);
+  EXPECT_EQ(u"Pin tab…", items[1].title);
```

Comment:
Tests will work well when translated in non-english locale?

Reply:
I think the tests for an en-US locale, so this should be okay
https://source.chromium.org/chromium/chromium/src/+/main:base/test/test_suite.cc;l=562?q=locale%20test%20f:.cc&ss=chromium